ts/Dashboards/Plugins/NavigatorComponent.ts

```diff
@@ -703,6 +756,12 @@ class NavigatorComponent extends Component {
                 ));
 
                 data = seriesData;
+                if (min !== void 0) {
```

Comment:
The rest of the code is clear but I would add a comment and describe this hack 😉